src/backend/aspen/api/views/phylo_trees.py

```diff
@@ -66,9 +66,12 @@ async def _get_selected_samples(db: AsyncSession, phylo_tree_id: int):
     # identifier check here since the process_phylo_tree method already does that
     # filtering, and this data is only used to match any identifiers that are
     # *already* on the tree.
+    gisaid_prefix = "hCoV-19/"
     for uploaded_pathogen_genome in phylo_run.inputs:
         sample = uploaded_pathogen_genome.sample
-        selected_samples.add(prefix_regex.sub("", sample.public_identifier))
+        stripped_identifier = prefix_regex.sub("", sample.public_identifier)
+        selected_samples.add(stripped_identifier)
+        selected_samples.add(f"{gisaid_prefix}{stripped_identifier}")
```

Comment:
so we're adding the idenitifers twice to the set of selected samples? Once with the gisaid_prefix stripped and then once more with the prefix added?

Reply:
it looks to me that `selected_samples` here are what samples the users selected, that will be later used to mark them accordingly in the sample id download, so here we include their public ID with or without gisaid_prefix and private ID (see line 75) to be comprehensive so regardless which ID we will use later for matching, it will not be missed. I think...